src/unix/bsd/apple/mod.rs

```diff
@@ -1580,6 +1580,50 @@ s_no_extra_traits! {
         pub ifr_name: [::c_char; ::IFNAMSIZ],
         pub ifr_ifru: __c_anonymous_ifr_ifru6,
     }
+
+    pub struct ifmibdata {
```

Comment:
Could these just go in `s!` rather than `s_no_extra_traits!` so the trait implementations don't need to be handwritten?

Reply:
Ah I was wondering why I needed to implement these traits myself. Completely forgot about the macro... ^^'